content/02.main-text.md

```diff
@@ -3,31 +3,21 @@
 This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
 However, the scientific community requires tools whose workflows encourage openness.
-Manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, by which anyone is able to contribute and the contribution history is preserved and public.
-We report a new tool and workflow for authoring scholarly manuscripts in the open, as well as the collaborative project that led to its creation.
-
-Open scholarly writing, a form of crowdsourcing [@pmcid:PMC4719068], has particular benefits for review articles, which present the state of the art in a scientific field [@doi:10.1371/journal.pcbi.1003149].
-Literature reviews are typically written in private by an invited team of colleagues.
-In contrast, broadly opening the process to anyone engaged in the topic --- such that planning, organizing, writing, and editing occur collaboratively in a public forum where anyone is welcome to participate --- can maximize a review's value.
-Open drafting of reviews is especially helpful for capturing state-of-the-art knowledge about rapidly advancing research topics at the intersection of existing disciplines where contributors bring diverse opinions and expertise.
-
-Based on our experience leading a recent open review [@tag:techblog_manubot], we discuss the pros and cons of open collaborative writing.
-Our review manuscript [@doi:10.1098/rsif.2017.0387], code-named the Deep Review, surveyed deep learning's role in biology and precision medicine, a research area undergoing explosive growth.
-In addition, we introduce [Manubot](https://github.com/greenelab/manubot-rootstock), the infrastructure we created to enable open manuscript writing online for the Deep Review, which was subsequently adopted by other projects.
+Manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, in which anyone is able to contribute and the contribution history is preserved and public.
+We introduce [Manubot](https://github.com/greenelab/manubot-rootstock), a new tool and infrastructure for authoring scholarly manuscripts in the open, and report how it was instrumental for the collaborative project that led to its creation.
 
+Based on our experience leading a recent open review [@tag:techblog_manubot], we discuss the advantages and challenges of open collaborative writing, a form of crowdsourcing [@pmcid:PMC4719068].
+Our review manuscript [@doi:10.1098/rsif.2017.0387] was code-named the Deep Review and surveyed deep learning's role in biology and precision medicine, a research area undergoing explosive growth.
```

Comment:
I don't personally have a problem with the language, but I wasn't an author on Deep Review, and a scrupulous person could object to using "our." I don't suggest any changes, but I just thought I'd mention it.

Reply:
I couldn't find specific examples, but I believe I have seen "our" used previously in the literature even when it only pertained to a subset of the authors.  An alternative is to be more specific:
> Based on our experience (DSH, CSG, AG) leading...

but that seems awkward to me.  I'm inclined to leave it as is unless any of us have a problem with this.

Having an author contribution statement could also help clarify which of us were involved in the Deep Review.  Does anyone want to add that?

If we do change these sentences, we'll have to propagate the updates to the other paragraphs that pertain to Deep Review as well.

Reply:
Leave as is, is fine with me.